sydent/config/crypto.py

```diff
@@ -33,40 +33,30 @@ def parse_config(self, cfg: "ConfigParser") -> bool:
         signing_key_str = cfg.get("crypto", "ed25519.signingkey")
         signing_key_parts = signing_key_str.split(" ")
 
-        save_key = False
-
         if signing_key_str == "":
-            logger.info(
-                "This server does not yet have an ed25519 signing key. "
-                "Creating one and saving it in the config file."
+            logger.warning(
+                "'ed25519.signingkey' cannot be blank. Please generate a new"
+                " signing key with the 'generate-key' script."
             )
 
             self.signing_key = signedjson.key.generate_signing_key("0")
 
-            save_key = True
+            return True
         elif len(signing_key_parts) == 1:
             # old format key
-            logger.info("Updating signing key format: brace yourselves")
+            logger.warning(
+                "Updating signing key format for this run. Please run the"
+                " 'update-key' script to speedup the next startup"
```

Comment:
Maybe mention that this feature will go away in the future?

Reply:
As in you want to drop support for the hex encoded keys?

Reply:
I assumed that's why we were making this change, but I don't really have the full context of why we want to do this.

Reply:
Oh right so currently, if it detects a hex encoded key, it will update it and then save the updated version to the config file. 

I'm removing the ability to overwrite an existing config file, but I don't want to remove the ability for people to update their keys, so I added a script and a log message that points you towards it.